packages/http/src/mocker/negotiator/responseFinder.ts

```diff
@@ -0,0 +1,52 @@
+import { IHttpOperationResponse } from '@stoplight/types';
+import { head } from 'fp-ts/lib/Array';
+import { flatten, fold, none, Option, some } from 'fp-ts/lib/Option';
+import { pipe } from 'fp-ts/lib/pipeable';
+import { tail } from 'lodash';
+import { Logger } from 'pino';
+import { createResponseFromDefault, findResponseByStatusCode } from './InternalHelpers';
+
+const msg = (code: number) => `Unable to find a ${code} response definition`;
+const defaultMsg = (code: number) => `Created a ${code} from a default response`;
```

Comment:
If not used at least 2-3 times, maybe let's not rush in create an external function.